lastminutewf2020.yml

```
@@ -1,5 +1,30 @@
 ---
 - hosts: all
+  tasks:
+     - name: Pre-generate the kernel flags for ansible usage
+       set_fact:
+         procline: "quiet ipv6.disable=1 cgroup_enable=memory swapaccount=1"
+    - name: Ensure /etc/default/grub has proper GRUB_CMDLINE_LINUX_DEFAULT
+      ansible.builtin.replace:
+        path: /etc/default/grub
+        regexp: '^GRUB_CMDLINE_LINUX_DEFAULT='
+        line: "GRUB_CMDLINE_LINUX_DEFAULT=\"{{ procline }}\""
+        owner: root
+        group: root
+        mode: '0644'
+      notify: update grub
+    - name: Ensure /etc/default/grub has proper GRUB_TIMEOUT
+      ansible.builtin.replace:
+        path: /etc/default/grub
+        regexp: "GRUB_TIMEOUT=.*"
+        replace: "GRUB_TIMEOUT=3"
+        owner: root
+        group: root
+        mode: '0644'
+      notify: update grub
+  handlers:
+    - name: update grub
+      shell: /usr/sbin/update-grub
```

```suggestion
      command: /usr/sbin/update-grub
      notify: reboot
    - name: reboot
      reboot:
```
If grub is updated we want to have a reboot to load the correct values, this will only happen at the end of the run unless the handlers are flushed earlier like we do for DOMjudge judgedaemons.

---

so if it seems like I should add the reboot to this?